### ISSUE TITLE
Crash by Access the Storage Request

### ISSUE THREAD

**Issue by TheCherry** [badasintended/slotlink#89]
**Setup**
- Minecraft: 1.16.5
- Fabric API: 0.35.0
- slotlink: 2.4.0
- related mod: ?

**Problem**
The Game-Crash then I access the Storage Request:
```[17:49:33] [main/FATAL]: Unreported exception thrown!
java.lang.IllegalAccessError: class badasintended.slotlink.client.gui.screen.RequestScreen$x$2 tried to access protected field net.minecraft.class_465.field_2776 (badasintended.slotlink.client.gui.screen.RequestScreen$x$2 and net.minecraft.class_465 are in unnamed module of loader net.fabricmc.loader.launch.knot.KnotClassLoader @479ceda0)
	at badasintended.slotlink.client.gui.screen.RequestScreen$x$2.get(RequestScreen.java:39) ~[slotlink-2.4.0+1.16.5.jar:?]
	at badasintended.slotlink.client.gui.screen.RequestScreen.getX(RequestScreen.java:39) ~[slotlink-2.4.0+1.16.5.jar:?]
	at svenhjol.charm.base.helper.ScreenHelper.getX(ScreenHelper.java:8) ~[charm-fabric-1.16.5-2.3.2.jar:?]
	at svenhjol.charm.client.InventoryTidyingClient.handleGuiSetup(InventoryTidyingClient.java:91) ~[charm-fabric-1.16.5-2.3.2.jar:?]
	at svenhjol.charm.event.GuiSetupCallback.lambda$static$0(GuiSetupCallback.java:14) ~[charm-fabric-1.16.5-2.3.2.jar:?]
	at net.minecraft.class_437.handler$zkp000$hookConstructor(class_437.java:1058) ~[intermediary-fabric-loader-0.11.5-1.16.5.jar:?]
	at net.minecraft.class_437.method_25423(class_437.java:326) ~[intermediary-fabric-loader-0.11.5-1.16.5.jar:?]
	at badasintended.slotlink.client.gui.screen.RequestScreen.method_25423(RequestScreen.java:282) ~[slotlink-2.4.0+1.16.5.jar:?]
	at net.minecraft.class_310.method_1507(class_310.java:922) ~[intermediary-fabric-loader-0.11.5-1.16.5.jar:?]
	at net.minecraft.class_3929$class_3930.method_17543(class_3929.java:65) ~[intermediary-fabric-loader-0.11.5-1.16.5.jar:?]
	at net.minecraft.class_3929.method_17541(class_3929.java:51) ~[intermediary-fabric-loader-0.11.5-1.16.5.jar:?]
	at net.minecraft.class_634.method_17587(class_634.java:1117) ~[intermediary-fabric-loader-0.11.5-1.16.5.jar:?]
	at net.minecraft.class_3944.method_17591(class_3944.java:42) ~[intermediary-fabric-loader-0.11.5-1.16.5.jar:?]
	at net.minecraft.class_3944.method_11054(class_3944.java:12) ~[intermediary-fabric-loader-0.11.5-1.16.5.jar:?]
	at net.minecraft.class_2600.method_11072(class_2600.java:21) ~[intermediary-fabric-loader-0.11.5-1.16.5.jar:?]
	at net.minecraft.class_1255.method_18859(class_1255.java:144) ~[intermediary-fabric-loader-0.11.5-1.16.5.jar:?]
	at net.minecraft.class_4093.method_18859(class_4093.java:23) ~[intermediary-fabric-loader-0.11.5-1.16.5.jar:?]
	at net.minecraft.class_1255.method_16075(class_1255.java:118) ~[intermediary-fabric-loader-0.11.5-1.16.5.jar:?]
	at net.minecraft.class_1255.method_5383(class_1255.java:103) ~[intermediary-fabric-loader-0.11.5-1.16.5.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1015) ~[intermediary-fabric-loader-0.11.5-1.16.5.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:681) [intermediary-fabric-loader-0.11.5-1.16.5.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:215) [intermediary-fabric-loader-0.11.5-1.16.5.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:234) [fabric-loader-0.11.5.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:153) [fabric-loader-0.11.5.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28) [fabric-loader-0.11.5.jar:?]
```
**Reproduction**
![image](https://user-images.githubusercontent.com/1090782/121924886-a121a280-cd3c-11eb-970b-697d8ba2cfc1.png)

**Remarks**
Modlist:
```
artifacts-3.2.1+fabric.jar
BetterMineshafts-Fabric-1.16.4-1.0.2.jar
bitsandchisels-2.3.2.jar
canvas-mc116-1.0.1511.jar
charm-fabric-1.16.5-2.3.2.jar
charmonium-fabric-1.16.5-2.3.2.jar
CraftPresence-Fabric-1.16.5-Release-1.8.2.jar
Economy-1.3.jar
fabric-api-0.35.0+1.16.jar
fabric-language-kotlin-1.6.1+kotlin.1.5.10.jar
farmers-delight-fabric-1.16.5-0.0.3.jar
flytre_lib-0.2.11.jar
gunpowder-base-0.5.7+1.16.5.jar
gunpowder-currency-1.0.2gunpowder.0.5.5.mc.1.16.5.jar
gunpowder-market-1.0.3+gunpowder.0.4.0.mc.1.16.5.jar
gunpowder-teleport-1.0.6+gunpowder.0.5.2.mc.1.16.5.jar
inventoryprofiles-fabric-1.16.2-0.4.2.jar
Mambience-3.1.1+1.16.5.jar
modmenu-1.16.9.jar
Patchouli-1.16.4-51-FABRIC.jar
pipe-1.3.1.jar
repurposed_structures-1.16.5-1.11.5-fabric.jar
slotlink-2.4.0+1.16.5.jar
Terra-fabric-5.3.3-BETA+5dd00db8-shaded-mapped.jar
trinkets-2.6.7.jar
YungsApi-1.16.5-Fabric-5.jar
```

**Comment by deirn** [comment]
can you try [this](https://github.com/badasintended/slotlink/suites/2996250817/artifacts/67713125) and see if it's fixed? use the jar with no `-sources` or `-dev` suffix

**Comment by TheCherry** [comment]
I can open the Storage now, but if I interact with (put a item into or try to take one out) the game like freeze. I can still move, place blocks, destroy block, but any another interaction and any mob stop from working.

**Comment by TheCherry** [comment]
OK. I removed the old blocks from before and placed new ones. Now its working.

**Comment by deirn** [comment]
alright, closing this

**Comment by deirn** [comment]
released [2.4.1](https://github.com/badasintended/slotlink/releases/tag/2.4.1)
